card/mod/01_core/set/all/tracked_attributes.rb

```diff
@@ -53,7 +39,7 @@ def extract_subcard_args! args
   subcards
 end
 
-def extract_type_id! args = {}
+def extract_type_id! args={}
```

Comment:
Assignment Branch Condition size for extract_type_id! is too high. [17.2/15]<br>Method has too many lines. [16/15]